docs/release-notes/1.2.0/release-1.2.0.md

```diff
@@ -0,0 +1,70 @@
+# [ML.NET](http://dot.net/ml) 1.2.0 
+## **General Availability**
+- **Microsoft.ML.TimeSeries**  
+    - Anomaly detection algorithms (Spike and Change Point):
+      - Independent and identically distributed.
+      - Singular spectrum analysis.
+      - Spectral residual from Azure Anomaly Detector/Kensho team.
+    - Forecasting models:
+      - Singular spectrum analysis.
+    - Prediction Engine for online learning
+      - Allows updating time series model with new observations at scoring so that the user does not have re-train the time series with old data each time.
+
+     [Samples](https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries)
+
+- **Microsoft.ML.OnnxTransformer**  
+   Allows scoring of ONNX models in the learning pipeline. Uses ONNX Runtime v0.4.
+      
+   [Sample](https://github.com/dotnet/machinelearning/blob/master/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs)
+
+- **Microsoft.ML.TensforFlow**  
+   Allows scoring of TensorFlow models in the learning pipeline. Uses TensforFlow v1.13. Very useful for image and text classification.
```

Comment:
"Enables" instead of "allows"

Perhaps clarify why it is very useful for image/text classifications. Does it enable them easier? Is it a community asked feature.